app/common/themes/GristLight.ts

```diff
@@ -130,7 +130,7 @@ export const GristLight: ThemeTokens = {
 
     /* Right Panel */
     rightPanelTabButtonHoverBg: tokens.primaryMuted,
-    rightPanelSubtabHoverFg: tokens.primaryMuted,
+    rightPanelSubtabFg: '#707070',
```

Comment:
this is a hard-coded value because the `secondary` gray on the default light theme doesn't match too well on panel backgrounds in terms of readability.

I feel it's better that way than sticking with the default gray, the small change makes the text noticeably more readable and this color code is already used elsewhere.